README.md

```diff
@@ -85,3 +85,41 @@ Do the following:
    * `go get github.com/some/lib@v1.2.3`: Get a particular version.
 2. Run `go mod tidy` to remove any unused modules.
 3. Run `go mod verify` to add packages needed for test packages etc.
+
+## Pulling the docker image
+
+Unfortunately the Github packages docker repo is not very "public", more on this can be found in this community [thread](https://github.community/t5/GitHub-Actions/docker-pull-from-public-GitHub-Package-Registry-fail-with-quot/td-p/32782/page/4). This means a docker login needs to be done before the images can be pulled. To do this follow these steps:
+
+1. Create a new token with the scope `read:packages` [here](https://github.com/settings/tokens).
+2. Save your token to e.g. to a file (or use environment variable or similar).
+3. Login with docker to `docker.pkg.github.com`.
+
+Using token saved to the file github.token: 
+
+```bash
+docker login -u yourgithubusername --password=$(cat github.token) docker.pkg.github.com
+```
+
+Using token in environmental variable GITHUB_TOKEN:
+
+```bash
+docker login -u yourgithubusername --password=$GITHUB_TOKEN docker.pkg.github.com
+```
+
+4. Pull the the docker image.
+
+The latest master version:
+
+```bash
+docker pull docker.pkg.github.com/qlik-oss/gopherciser/gopherciser:latest
+```
+
+Specific released version:
+
+```bash
+docker pull docker.pkg.github.com/qlik-oss/gopherciser/gopherciser:0.4.10
+```
+
+### Using the image in Kubernetes
+
+To use the image in Kubernetes, e.g. to perform executions as part of a Kubernetes job, credentials the Github package registry needs to be added the same way a private registry is used, see documentation [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
```

Comment:
"creadentials the Github package" - is a word missing in there?
Change ..."registry needs"... to ..."registry need"...?